connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -336,3 +347,31 @@ export async function webCrawlerGarbageCollector(
     }
   } while (foldersToDelete.length > 0);
 }
+
+export async function getWebsitesToCrawl() {
+  const frequencyToSQLQuery: Record<CrawlingFrequency, string> = {
+    never: "never",
+    daily: "1 day",
+    weekly: "1 week",
+    monthly: "1 month",
+  };
+  const allConnectorIds: ModelId[] = [];
+
+  for (const frequency of CrawlingFrequencies) {
+    if (frequency === "never") {
```

Comment:
We can probably omit it from the `frequencyToSQLQuery` then?

Reply:
Typescript is not happy when I do that.